### PR TITLE
Degeneracy

### PR DESCRIPTION
**Pull request by FrownTown**
## About The Pull Request
Re-implements degenerate socks and underwear that were removed from the game previously because they wouldn't properly fit bodytypes. 

Also implemented one tank top until I can figure out how to make it differentiate between the gendered shirt varieties without having to split the .dmi up.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -462,7 +462,7 @@ var/global/list/damage_icon_parts = list()
 		"belt"			= 'icons/inventory/belt/mob.dmi',
 		"s_store"		= 'icons/inventory/on_suit/mob.dmi',
 		"backpack"		= 'icons/inventory/back/mob.dmi',
-		"underwear"		= 'icons/inventory/underwear/mob.dmi'
+		"underwear"		= 'icons/mob/human_races/underwear.dmi'
```
Comment (Kurgis) [inline code/modules/mob/living/carbon/human/update_icons.dm]: Why? This should remain in inventory.

**Comment by drexample** [comment]
Disgusting! Repulsive! You didn't post the pictures!

**Comment by mattroks101** [comment]
:mistake:

**Comment by FrownTown** [comment]
This PR is now complete. Color blending is fixed, added most of the sprites, also made the undershirts automatically switch between male and female sprite variants based on the character's gender. Left out just a few sprites that I felt were incredibly ugly or immersion breaking.

**Comment by FrownTown** [comment]
P.S. also moved the icon files back into inventory/underwear/mob and mob_fem.dmi

**Comment by PlinHost** [comment]
There are some layer issues: bras are drawn over undershirts and socks are drawn over underwear.
img

**Comment by FrownTown** [comment]
> There are some layer issues: bras are drawn over undershirts and socks are drawn over underwear.
> img

Socks drawing over underwear is intended (you wear pantyhose over underwear, obviously, not the other way around... unless you're a superhero)

unfortunately I can't figure out how to fix the issue with bras and undershirts, so I'm just removing undershirts from the PR. It should be good to go, I guess, minus tanktops.

**Pull request closed without merging**